### PR TITLE
 Move udf exception handling tests from tinc to ICW.

### PR DESCRIPTION
**Pull request by ashwinstar**
Author: Ashwin Agrawal <aagrawal@pivotal.io>
Date:   Tue Feb 6 09:11:09 2018 -0800

    Move udf exception handling tests from tinc to ICW.

    Simplified the tests by setting the GUCs at session level instead of
    reloads. Also removed creating tables with index, sequence etc as not
    needed for these tests.

    So, same coverage is achieved with ICW in 15 secs vs older version
    taking 15 mins in TINC.

Author: Ashwin Agrawal <aagrawal@pivotal.io>
Date:   Tue Feb 6 13:33:44 2018 -0800

    TINC: Delete make target aocoalter_catalog_loaders.

    Delete udf_exception_handling tinc tests.

    Also remove tinc stage from CI `aocoalter_catalog_loaders`. Majority
    of tests from this group has been moved to ICW and remaining ones are
    now part of generic `make storage` target.

### CODE REVIEW

File: src/test/regress/sql/udf_exception_handling.sql [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+ END;
+END;
+$$
+LANGUAGE plpgsql;
+SET debug_dtm_action_segment=0;
+SET debug_dtm_action_target=protocol;
+SET debug_dtm_action_protocol=subtransaction_rollback;
+SET debug_dtm_action=fail_end_command;
+SET debug_dtm_action_nestinglevel=0;
+-- @Description: The Test executes Procedure test_setup_1 to exercise UDF Exception Handling in various subtransaction condition for debug_dtm_action_target=protocol and debug_dtm_action_segment = 0
```
Comment (asimrp) [inline src/test/regress/sql/udf_exception_handling.sql]: All the `@Description`s are identical, can we please keep only one copy?

File: src/test/tinc/tincrepo/mpp/gpdb/tests/catalog/udf_exception_handling/test_udf_exception_handling.py [hunk 2]
```diff
@@ -1,155 +0,0 @@
...
-
-    def test_UDF_exception(self):
-        ''' 
-        
-        @param debug_dtm_action_segment 
-        @param debug_dtm_action_target 
-        @param debug_dtm_action_protocol  or debug_dtm_action_sql_command_tag
-        @param debug_dtm_action 
-        @param debug_dtm_action_nestinglevel
-        @description: This tests the Exception Handling of GPDB PL/PgSQL UDF
```
Comment (asimrp) [inline src/test/tinc/tincrepo/mpp/gpdb/tests/catalog/udf_exception_handling/test_udf_exception_handling.py]: This long comment seems helpful, can we please keep it in one of the ICW .sql files?

File: concourse/pipelines/templates/gpdb-tpl.yml [hunk 3]
```diff
@@ -1720,18 +1720,10 @@ jobs:
       passed: [gate_cs_start]
       trigger: [[ test_trigger ]]
     - get: centos-gpdb-dev-6
-  - aggregate:
-    - task: aocoalter_catalog_loaders
```
Comment (asimrp) [inline concourse/pipelines/templates/gpdb-tpl.yml]: :+1: Thoughtful change this is, saves container setup, demo cluster creation, etc.

**Comment by hlinnaka** [comment]
Great, thanks!

File: src/test/tinc/tincrepo/mpp/gpdb/tests/catalog/udf_exception_handling/test_udf_exception_handling.py (hunk 2 above)
Comment (ashwinstar) [reply src/test/tinc/tincrepo/mpp/gpdb/tests/catalog/udf_exception_handling/test_udf_exception_handling.py]: Thanks added.

**Pull request merged**